x/auth/ante/ante_test.go

```diff
@@ -482,7 +482,8 @@ func (suite *AnteTestSuite) TestAnteHandlerFees() {
 		{
 			"signer as enough funds, should pass",
 			func() {
-				accNums = []uint64{7}
+				accNums = []uint64{acc1.GetAccountNumber()}
```

Comment:
I think when I removed IBC registration from simapp, one less account was being created causing the account number to become 6 and not 7. I used the account associated with `addr0` since that is the account being funded in this test